calico-cloud/image-assurance/understanding-scan-results.mdx

```diff
@@ -11,7 +11,32 @@ In the left navbar in Manager UI, click **Image Assurance**, **All Scanned Image
 ## All Scanned Images tab
 
 This tab lists scanned images if you have enabled or used one of the [Image Assurance scanners](./scanners/overview).
-To manage your scan results, you can filter the list results or delete particular scan result items.
+
+#### Filter Scan Results
+
+You can use different combinations of filters to refine your results, making it easier to monitor and focus on relevant vulnerabilities for your remediation efforts.
+
+#### Delete Scan Results
+
+If a scan result is no longer relevant to your remediation effort, you can delete it by selecting multiple results and choosing the **Delete** option from the Bulk Actions dropdown.
+
+:::note
+
+**Deleted Results**: Deleting a scan result removes it from the **All Scanned Images** tab. However, if the image is currently running, its record will still appear in the **Running Images** tab, with the number of instances tracked.
+
+:::
+
+#### Dismiss/Restore Scan Results Temporarily
+
+If you want to temporarily hide scan results that are not important at the moment (e.g., "noise" or less critical vulnerabilities), you can dismiss them by selecting the rows and choosing the **Dismiss** option from the Bulk Actions dropdown.
+
+Dismissed results can be accessed at any time for future review by enabling the **Dismissed** toggle filter. To restore them you can select the rows and choose the **Restore** option from the Bulk Actions dropdown.
+
+:::note
+
+**Dismissed Results**: While a scan result is dismissed, it will have no impact on the Admission Controller policies.
+
+:::
```

Comment:
Ambiguous. What do you mean?
* Even if you dismiss a scan result, that result will continue to affect the Admission Controller policies.
* If you dismiss a scan result, the Admission Controller policies will no longer be affected by that scan result.

Reply:
"If you dismiss a scan result, the Admission Controller policies will no longer be affected by that scan result."
But this can be omitted as users likely expect this behavior.